modules/install/mod_settings/db.xml

```diff
@@ -116,7 +116,7 @@
                 <type>datetime</type>
                 <null></null>
                 <key></key>
-                <default>0</default>
+                <default></default>
```

Comment:
Can you elaborate a bit why removing the default is necessary here? (same question applies to the other datetime fields)